index.js

```diff
@@ -85,6 +85,23 @@ server.delete('/api/cohorts/:id', async (req, res) => {
   }
 });
```

Comment:
same thing for the put, if the name value is missing, have it return specified status code and error message